src/components/DropdownMenu/README.md

```diff
@@ -528,7 +528,8 @@ This type describes individual dropdown menu items.
 | :----------- | :------------------------------------------------------------------------------------------------------------------------------ | :--------------------------------------------: | :-----: |
 | `text`       | Menu item content.                                                                                                              |               `React.ReactNode`                |         |
 | `action`     | Menu item click handler. Recieves the parameters from the parent dropdown menu component (both `event` and `data`).             | `(event: React.MouseEvent, data: any) => void` |         |
-| `icon`       | Menu item icon.                                                                                                                 |               `React.ReactNode`                |         |
+| `iconStart`  | Menu item icon before item text.                                                                                                |               `React.ReactNode`                |         |
+| `iconEnd`    | Menu item icon after item text. Ignored, when item has sub-menu.                                                                |               `React.ReactNode`                |         |
```

Comment:
&rarr; 'Menu item icon after the item content. Ignored if the item has a submenu.'